### DISCUSSION TITLE
Is there a roadmap for remult?

### DISCUSSION THREAD

**Discussion by ghost** [noam-honig/radweb#217] [General]
Is there a roadmap for remult? 
I want to know more.

**Comment by yoni-rapoport** [comment]
We're pretty impatient when it comes to adding features to Remult, so whenever we see something worth putting into the roadmap, we pretty much go ahead and implement it instead...

Our main goal is reaching a stable (v1) API, which still requires more community feedback to finalize. In parallel, we have a lot of work to do on documentation, both in terms of completeness as well as making it easier to use.

At some point, we may try to add deeper integration with generic CRUD UI components because we still see a lot of repetitive coding around "binding" entity data and operations to (eventually) DOM elements and events.

We're always open to feedback and feature requests so if you have any suggestions for the roadmap please let us know.
